### PR TITLE
UI: Fix build errors

### PR DESCRIPTION
**Pull request by ordabach**
# What does this PR do?

Fix build errors

## PR Checklist
* [ ] Have you added an explanation of what your changes do and why you'd like to include them?
* [ ] Is the TravisCI build passing?
* [ ] Was the CHANGELOG.md updated to reflect the changes?
* [ ] Was the documentation framework updated to reflect the changes?
* [ ] Have you checked that you haven't introduced any duplicate code?

## Testing Checklist

* [ ] Added relevant unit tests?
* [ ] Do all unit tests pass?
* [ ] Do all end-to-end tests pass?
* [ ] Any other testing performed?
    > Tested by {Running the Monkey locally with relevant config/running Island/...}
* [ ] If applicable, add screenshots or log transcripts of the feature working

### CODE REVIEW

**Review by mssalvatore — COMMENTED** [review]
After rebasing on develop, I get 
image

File: monkey/monkey_island/cc/ui/src/components/ui-components/MonkeySelect.tsx [hunk 1]
```diff
@@ -26,7 +26,7 @@ type SelectProps = MUISelectProps & {
   options: Option[],
   selectedOption: Option,
   onChange: (event: SelectChangeEvent) => void,
-  variant?: SelectVariant
+  variant?: string
```
Comment (VakarisZ) [inline monkey/monkey_island/cc/ui/src/components/ui-components/MonkeySelect.tsx:29]: Doesn't make any sense, should be a proper way to fix this

**Comment by mssalvatore** [comment]
I still get this when using `npm start`

image

**Comment by VakarisZ** [comment]
Merged in 7dca764

**Pull request closed without merging**